netbox/project-static/css/base.css

```diff
@@ -62,8 +62,20 @@ footer p {
     }
 }
 
+/* Scroll the drop-down menus at or above 768px wide to match bootstrap's behavior for hiding dropdown menus */
+@media (min-width: 768px) {
+    .navbar-nav>li>ul {
+        max-height: 80vh;
```

Comment:
80vh does have an issue once you get to 150px but I don't believe it is anything to worry about really, just more of a FYI.

Reply:
The root of the issue is that I'm not subtracting the header height like so
```
max-height: calc(80vh - 50px);
```

If we're confident that we won't be changing the height of the header anytime soon, I'm fine with hardcoding `50px`. Preferably, I'd like to add this in.

Reply:
I've gone and added the above. If anyone objects to hard-coding 50px, speak up. [previously](https://github.com/netbox-community/netbox/blob/98de88de901f5eab752fd1f93cd94a08eb73d281/netbox/project-static/js/forms.js#L396), I've opted to dynamically calculating the header height, but that's not possible in CSS.